### PR TITLE
Added setting to display external search link. Issue 195.

### PR DESCRIPTION
**Pull request by MikaelNordberg**

### CODE REVIEW

File: PXWeb/Resources/Styles/scss/pxweb/_PxWeb.scss [hunk 1]
```diff
@@ -148,6 +148,28 @@
   padding-left: 1px;
 }
 
+
+
+#pxwebcontent .px-external-search-div {
+    margin: 20px;
+    margin-right: 30px;
```
Comment (JohannesFinsveen) [inline PXWeb/Resources/Styles/scss/pxweb/_PxWeb.scss:155]: The two others have #header, is it left out on purpose here?

File: PXWeb/Resources/Languages/pxlang.xml [hunk 2]
```diff
@@ -1323,6 +1325,8 @@
    <sentence name="PxWebAdminSettingsMainHeaderForTablesTableName" value="Table name"/> 
    <sentence name="PxWebAdminSettingsMainHeaderForTablesStatisticArea" value="Statistics area"/>
    <sentence name="PxWebAdminSettingsMainHeaderForTablesInfo" value="Type of heading that will be used for table pages"/>
+   <sentence name="PxWebAdminSettingsShowExternalSearchLink" value="Show external search"/>
+   <sentence name="PxWebAdminSettingsShowExternalSearchLinkInfo" value="If link to external search page shall be displayed or not"/>
```
Comment (JohannesFinsveen) [inline PXWeb/Resources/Languages/pxlang.xml:1329]: Should we add something like: Typically the big search on your site ?

**Review by JohannesFinsveen — COMMENTED** [review]
2 small comments :-)  
(Built it on http://pxwebmaster.azurewebsites.net/pxweb/en/Example/ , so it runs with the old config :-) )

File: PXWeb/Resources/Styles/scss/pxweb/_PxWeb.scss (hunk 1 above)
Comment (MikaelNordberg) [reply PXWeb/Resources/Styles/scss/pxweb/_PxWeb.scss:155]: Actually it is a copy of the style for px-change-lang-div. Maybe we should add header to both of them (px-external-search-div and px-change-lang-div)?

File: PXWeb/Resources/Languages/pxlang.xml (hunk 2 above)
Comment (MikaelNordberg) [reply PXWeb/Resources/Languages/pxlang.xml:1329]: That is a good idea

File: PXWeb/Resources/Styles/scss/pxweb/_PxWeb.scss (hunk 1 above)
Comment (JohannesFinsveen) [reply PXWeb/Resources/Styles/scss/pxweb/_PxWeb.scss:155]: Can't see why not :-)

**Review by likp — APPROVED** [review]

**Pull request merged**